test/data/speedreader/article/csp_http.html.mock-http-headers

```diff
@@ -0,0 +1,4 @@
+HTTP/1.1 200 OK
+Content-Type: text/html;
+Content-Security-Policy default-src 'none';img-src 'none';style-src 'none';frame-ancestors 'none';form-action 'none';sandbox
```

Comment:
missing `:` after Content-Security-Policy?

Reply:
also `img-src 'none'` and `style-src 'none'` are unnecessary because default-src is none.

Reply:
> also `img-src 'none'` and `style-src 'none'` are unnecessary because default-src is none.
Yes, but I want the tests to produce the same console output.